### PR TITLE
PTRef: Increase weight of route<->SP link to avoid unwanted shortcuts

### PR DESCRIPTION
**Pull request by pbougue**
JIRA: https://jira.kisio.org/browse/NAVITIAII-2793

fix after #2931

### CODE REVIEW

**Review by kinnou02 — APPROVED** [review]

**Review by nonifier — APPROVED** [review]

File: source/type/tests/comments_test.cpp [hunk 1]
```diff
@@ -50,7 +50,7 @@ BOOST_GLOBAL_FIXTURE(logger_initialized);
 
 namespace navitia {
 namespace type {
-bool operator==(const std::vector<Comment>& lhs, const std::vector<std::string>& rhs) {
+static bool operator==(const std::vector<Comment>& lhs, const std::vector<std::string>& rhs) {
```
Comment (benoit-bst) [inline source/type/tests/comments_test.cpp:53]: Why this change ?

**Review by benoit-bst — APPROVED** [review]

File: source/type/tests/comments_test.cpp (hunk 1 above)
Comment (pbougue) [reply source/type/tests/comments_test.cpp:53]: Just remove a clang warning, as it's missing prototype.
